### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]:NVDA is not announicng appropriately Static Text present in Browse Repositories Window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1504]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Navigate and select Settings Menu item under File menu.
3. In Settings dialog  box, navigate and select Plugins, and then Select Browse repositories  button.
4. In Browse Repositories window , Press "Ctrl+Tab" then press tab t to access the static text then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior**: NVDA Should announce  as AccuRev is the leading provider of process-centric software configuration management (SCM) with a best-of-breed approach to application lifecycle management (ALM). This approach enables unlimited, adaptable, and compliant process models, offering customers maximum flexibility and control.
Vendor
AccuRev Inc. http://www.accurev.com/ support@accurev.com
Plugin homepage
http://www.accurev.com/accubridge.html
Size
726.5 K for static text present in browse Repositories window

**Actual Result:**
**NVDA Behavior:** NVDA is announicng as 
v1.0.3
edit  text/html
v1.0.3
tab
edit  text/html
blank
**Jaws Behavior:** Jaws is announcing as expected
v2017
Use your reading keys to read the text.
text/html read only edit
AccuRev is the leading provider of process-centric software configuration management (SCM) with a best-of-breed approach to application lifecycle management (ALM). This approach enables unlimited, adaptable, and compliant process models, offering customers maximum flexibility and control.
Vendor
AccuRev Inc. http://www.accurev.com/ support@accurev.com
Plugin homepage
http://www.accurev.com/accubridge.html
Size
726.5 K
Use your reading keys to read the text.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![browse repositories static text](https://user-images.githubusercontent.com/22956130/36142489-4171d590-10ce-11e8-91bf-bc7e06c17e72.jpg)
![browse repositories-static](https://user-images.githubusercontent.com/22956130/36142409-062d64d6-10ce-11e8-9e4e-229eac424431.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope.

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749938
